scripts/docs/update_web_vep_plugins_documentation.pl

```diff
@@ -263,7 +263,10 @@ sub read_plugin_file {
         }
         else {
           $line =~ s/^\s+//;
-          push(@developer, $1) if ($line =~ /(.+)\s+</);
+          if ($line =~ /(.+)\s+</) {
+            # some plugin had old-style Ensembl contact info with lots of text
+            push @developer, $1 unless (grep /^$1$/, @developer || $1 eq "developers list at");
```

Comment:
This will not work as expected unless you put parentheses to `grep`:

```suggestion
            push @developer, $1 unless (grep(/^$1$/, @developer) || $1 eq "developers list at");
```